util/Radiance_Monitor/image_gen/src/radmon_ig_summary.fd/summary.f90

```diff
@@ -15,7 +15,9 @@ program summary
 !
 !************************************************************************
 
-   use IFPORT
+!   use IFPORT
+! Intel® Fortran includes functions and subroutines that ease porting of code to or from a PC, or allow you to write code on a PC that is compatible with other platforms.  The portability library is called LIBIFPORT.LIB (Windows*) or libifport.a (Linux* and macOS*). Frequently used functions are included in a portability module called IFPORT.
+! What functionality from IFPORT library is being used here and why?
```

Comment:
I don't believe there is any need for IFPORT here.  Best guess is that it's a result of cutting and pasting some years ago.  I'll add evaluating it for removal to my to do list.